src/desktoputils.h

```diff
@@ -26,6 +26,9 @@ namespace tremotesf::desktoputils {
     };
     const QIcon& statusIcon(StatusIcon icon);
 
+    enum PriorityIcon { HighPriorityIcon, NormalPriorityIcon, LowPriorityIcon };
```

Comment:
Better use enum class, and remove *Icon suffixes (ignore StatusIcon, I will change that to enum class later).